presto-tpch/src/main/java/com/facebook/presto/tpch/TpchConnectorFactory.java

```diff
@@ -36,6 +36,7 @@
 
     private final int defaultSplitsPerNode;
     private final boolean predicatePushdownEnabled;
+    private final boolean partitioningEnabled;
```

Comment:
And why do we need to have unpartitioned tables?

Reply:
Because we want to emulate some realistic benchmark environment (so that each change doesn't require re-benchmarking if it doesn't change the plans).
For the benchmarks, we went with unpartitioned tables, so needed to disable partitioning in tpch connector used for the simulatation.
If we had benchmarks on partitioned tables, we would need to add _proper_ partitioning in the tpch connector. The existing one was purely opportunistic, not quite how the real data would be partitioned.